index.html

```diff
@@ -60,6 +58,87 @@ <h2 class="title text-primary">I’m a software developer</h2>
+                            <button type="button">html</button>
+                        </div>
+                        <div class="see-project">
+                            <button>See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="img-patholder">
+                    <div class="white-box">
+                        <div class="title 2">Multi-Post Stories <br>Gain+Glory</div>
+                        <div class="buttons">
+                            <button type="button">ruby on rails</button>
+                            <button type="button">Css</button>
+                            <button type="button">javascript</button>
+                            <button type="button">html</button>
+                        </div>
+                        <div class="see-project">
+                            <button>See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="img-patholder">
+                    <div class="white-box">
+                        <div class="title 2">Multi-Post Stories <br>Gain+Glory</div>
+                        <div class="buttons">
+                            <button type="button">ruby on rails</button>
+                            <button type="button">Css</button>
+                            <button type="button">javascript</button>
+                            <button type="button">html</button>
+                        </div>
+                        <div class="see-project">
+                            <button>See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="img-patholder">
+                    <div class="white-box">
+                        <div class="title 2">Multi-Post Stories <br>Gain+Glory</div>
+                        <div class="buttons">
+                            <button type="button">ruby on rails</button>
+                            <button type="button">Css</button>
+                            <button type="button">javascript</button>
+                            <button type="button">html</button>
+                        </div>
+                        <div class="see-project">
+                            <button>See project</button>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="title about-me">
+                <h2 class="title">About Me</h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <div class="get-my-resume">
+                    <button>Get My Resume</button>
+                </div>
```

Comment:
- [ ] Please notice the space hear between the button and first card is much greater than what you currently have. please increase this small area to much the Figma. It also makes more sense because you want there to be a clearly distinction between these two sections .

|Your Design | Figma Design |
-|-
|![image](https://user-images.githubusercontent.com/18730106/153635939-4ec9e5ae-5b54-49a0-a2ab-02eca5841c80.png)|![image](https://user-images.githubusercontent.com/18730106/153635978-17252a50-594b-4f8f-b1b6-61b272accadb.png)|